### PR TITLE
Add chance of loot to roundstart toilets

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Occasionally toilets can have random stuff in the cistern, at the
start of the shift.
/:cl:

Before, if you find an item in a toilet, someone deliberately hid it
there. Now there's a slim chance that maybe that welding torch in the
brig toilet was there through dumb luck, and not an inside job.

Constructed toilets do not come with a chance of maintenance loot.

Sanity checks are present to ensure that something isn't spawned inside
the cistern that you wouldn't be able to put back, and the
`spawn_loot()` proc is seperate, in case for some reason an admin wants
to call it repeatedly. (Did you know you could hide more than one item
in a cistern?)

### CODE REVIEW

**Comment by nightred** [comment]
This would give loot items to the perma brig toilets, is that an intended side effect?

**Comment by coiax** [comment]
> 
> 
> This would give loot items to the perma brig toilets, is that an intended side effect?

Yeah, I did mention it in the PR body. 😅

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -12,11 +12,29 @@
 	var/buildstacktype = /obj/item/stack/sheet/metal //they're metal now, shut up
 	var/buildstackamount = 1
 
+	// Chance that there's something hanging around in the cistern
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/structures/watercloset.dm:15]:
```suggestion
	/// Chance that there's something hanging around in the cistern
```

File: code/game/objects/structures/watercloset.dm [hunk 2]
```diff
@@ -12,11 +12,29 @@
...
+
 /obj/structure/toilet/Initialize()
 	. = ..()
 	open = round(rand(0, 1))
 	update_icon()
 
+	if(prob(loot_chance))
+		spawn_loot()
+
+/obj/structure/toilet/proc/spawn_loot()
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/structures/watercloset.dm:26]: b

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
You have generated new variables/procs/defines without proper documentation for the autodoc to work.

Please click [here ](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)for a guide on how to properly document these items.

**Comment by Krysonism** [comment]
This makes the cistern a very unreliable hiding place, because it increases the chance someone will search the toilet 4noraisin.

**Comment by anconfuzedrock** [comment]
the man who wastes time crowbarring a toilet for maint loot is not a smart one.
~the guy who wrote the art of war

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Cobby

**Comment by ExcessiveUseOfCobblestone** [comment]
i have no(t came to an) opinion on  the balance of this

**Pull request closed without merging**